io.gitlab.Turtlico.json

```
@@ -29,79 +33,85 @@
     ],
     "modules": [
         {
-            "name" : "tcl",
-            "sources" : [
+            "name": "tkinter",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --prefix=${FLATPAK_DEST} ."
+            ],
+            "sources": [
                 {
-                    "type" : "archive",
-                    "url" : "http://prdownloads.sourceforge.net/tcl/tcl8.6.10-src.tar.gz",
-                    "sha256" : "5196dbf6638e3df8d5c87b5815c8c2b758496eb6f0e41446596c9a4e638d87ed"
+                    "type": "git",
+                    "url": "https://github.com/iwalton3/tkinter-standalone",
+                    "commit": "ba946536054f9d27a08aafde21aa18330ce05729"
                 }
             ],
-            "subdir" : "unix",
-            "config-opts": [
-                "--enable-threads"
-            ],
-            "post-install" : [
-                "chmod 755 /app/lib/libtcl8.6.so"
-            ]
-        },
-        {
-            "name" : "tk",
-            "sources" : [
+            "modules": [
                 {
-                    "type" : "archive",
-                    "url" : "http://prdownloads.sourceforge.net/tcl/tk8.6.10-src.tar.gz",
-                    "sha256" : "63df418a859d0a463347f95ded5cd88a3dd3aaa1ceecaeee362194bc30f3e386"
+                    "name": "tcl",
+                    "buildsystem": "autotools",
+                    "subdir": "unix",
+                    "post-install": [
+                        "chmod 755 /app/lib/libtcl*.so"
+                    ],
+                    "cleanup": [
+                        "/bin",
+                        "/lib/pkgconfig",
+                        "/man"
+                    ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://prdownloads.sourceforge.net/tcl/tcl8.6.12-src.tar.gz",
+                            "sha256": "26c995dd0f167e48b11961d891ee555f680c175f7173ff8cb829f4ebcde4c1a6",
+                            "x-checker-data": {
+                                "type": "anitya",
+                                "project-id": 4941,
+                                "stable-only": true,
+                                "url-template": "https://prdownloads.sourceforge.net/tcl/tcl$version-src.tar.gz"
+                            }
+                        }
+                    ]
+                },
+                {
+                    "name": "tck",
```

```suggestion
                    "name": "tk",
```

it doesn't break anything, it's just a confusing typo